### PR TITLE
ee9 update for wsscxf.1 fat

### PR DESCRIPTION
**Pull request by jimmy1wu**
for https://github.com/OpenLiberty/open-liberty/issues/17688

### CODE REVIEW

**Comment by jimmy1wu** [comment]
#libby
#build

**Comment by jimmy1wu** [comment]
#libby
#build

File: dev/com.ibm.ws.wssecurity_fat.wsscxf.1/fat/src/com/ibm/ws/wssecurity/fat/cxf/wss11sig/CxfWss11SigTests.java [hunk 1]
```diff
@@ -308,7 +308,7 @@ public void testCXFClientBasicEncryptedElementMisMatchEE7Only() throws Exception
 
     @Test
     @SkipForRepeat(SkipForRepeat.NO_MODIFICATION)
-    @AllowedFFDC(value = { "org.apache.wss4j.common.ext.WSSecurityException", "java.net.MalformedURLException" }, repeatAction = { EE8FeatureReplacementAction.ID })
+    @AllowedFFDC(value = { "java.net.MalformedURLException" }, repeatAction = { EE8FeatureReplacementAction.ID })
```
Comment (meiaus) [inline dev/com.ibm.ws.wssecurity_fat.wsscxf.1/fat/src/com/ibm/ws/wssecurity/fat/cxf/wss11sig/CxfWss11SigTests.java:311]: The exception `"org.apache.wss4j.common.ext.WSSecurityException"` is missed in @AllowedFFDC? or it's not needed?  This exception is usually used for EE8.

File: dev/com.ibm.ws.wssecurity_fat.wsscxf.1/fat/src/com/ibm/ws/wssecurity/fat/cxf/x509token/CxfX509OverRideTests.java [hunk 2]
```diff
@@ -129,7 +129,7 @@ public static void setUp() throws Exception {
     //
 
     @Test
-    @AllowedFFDC(value = { "org.apache.wss4j.common.ext.WSSecurityException", "java.net.MalformedURLException" }, repeatAction = { EE8FeatureReplacementAction.ID })
+    @AllowedFFDC(value = { "java.net.MalformedURLException" }, repeatAction = { EE8FeatureReplacementAction.ID })
```
Comment (meiaus) [inline dev/com.ibm.ws.wssecurity_fat.wsscxf.1/fat/src/com/ibm/ws/wssecurity/fat/cxf/x509token/CxfX509OverRideTests.java:132]: The exception `"org.apache.wss4j.common.ext.WSSecurityException"` is missed? or it's not needed? This exception is usually used for EE8.

**Review by meiaus — CHANGES_REQUESTED** [review]
2 places to recheck. Thanks.

File: dev/com.ibm.ws.wssecurity_fat.wsscxf.1/fat/src/com/ibm/ws/wssecurity/fat/cxf/wss11sig/CxfWss11SigTests.java (hunk 1 above)
Comment (jimmy1wu) [reply dev/com.ibm.ws.wssecurity_fat.wsscxf.1/fat/src/com/ibm/ws/wssecurity/fat/cxf/wss11sig/CxfWss11SigTests.java:311]: the FFDC is not needed
i ran the test without it and it still passed
only the `java.net.MalformedURLException`  FFDC was thrown for that test

File: dev/com.ibm.ws.wssecurity_fat.wsscxf.1/fat/src/com/ibm/ws/wssecurity/fat/cxf/x509token/CxfX509OverRideTests.java (hunk 2 above)
Comment (jimmy1wu) [reply dev/com.ibm.ws.wssecurity_fat.wsscxf.1/fat/src/com/ibm/ws/wssecurity/fat/cxf/x509token/CxfX509OverRideTests.java:132]: ditto

File: dev/com.ibm.ws.wssecurity_fat.wsscxf.1/fat/src/com/ibm/ws/wssecurity/fat/cxf/wss11sig/CxfWss11SigTests.java (hunk 1 above)
Comment (meiaus) [reply dev/com.ibm.ws.wssecurity_fat.wsscxf.1/fat/src/com/ibm/ws/wssecurity/fat/cxf/wss11sig/CxfWss11SigTests.java:311]: ok thanks for clarification.

File: dev/com.ibm.ws.wssecurity_fat.wsscxf.1/fat/src/com/ibm/ws/wssecurity/fat/cxf/x509token/CxfX509OverRideTests.java (hunk 2 above)
Comment (meiaus) [reply dev/com.ibm.ws.wssecurity_fat.wsscxf.1/fat/src/com/ibm/ws/wssecurity/fat/cxf/x509token/CxfX509OverRideTests.java:132]: ok thanks for clarification.

**Review by meiaus — APPROVED** [review]

**Comment by arunavemulapalli** [comment]
#build

**Comment by jimmy1wu** [comment]
#build

**Comment by jimmy1wu** [comment]
#build

**Comment by arunavemulapalli** [comment]
for #12449

**Pull request merged**